extension/src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase/index.tsx

```diff
@@ -68,10 +63,6 @@ export const ConfirmMnemonicPhrase = ({
   const joinSelectedWords = () =>
     selectedWords.map((word) => convertToWord(word)).join(" ");
 
-  if (applicationState === APPLICATION_STATE.MNEMONIC_PHRASE_CONFIRMED) {
```

Comment:
this never gets reached because of the [parent component ](https://github.com/stellar/freighter/blob/d56499902dff206460f1615b5d75046645a53eaf/extension/src/popup/views/MnemonicPhrase.tsx#L22-L24)